Lib/multiprocessing/managers.py

```diff
@@ -1262,8 +1262,12 @@ class SharedMemoryServer(Server):
 
         def __init__(self, *args, **kwargs):
             Server.__init__(self, *args, **kwargs)
+            address = self.address
+            # The address of Linux abstract namespaces can be bytes
+            if isinstance(address, bytes):
+                address = address.decode()
             self.shared_memory_context = \
-                _SharedMemoryTracker(f"shmm_{self.address}_{getpid()}")
+                _SharedMemoryTracker(f"shmm_{address}_{getpid()}")
```

Comment:
I know "shm" (shared memory), but what does shmm (two M) stands for?

Reply:
The two 'm' in `memory`? I don't know :S

Reply:
Maybe removed the second M, it looks like a typo.

Reply:
Done